src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -699,9 +679,22 @@ internal override LocalSymbol GetAdjustedLocalSymbol(LocalSymbol local, int posi
             return adjustedLocal;
         }
 
-        private LocalFunctionSymbol GetDeclaredLocalFunction(LocalFunctionStatementSyntax declarationSyntax, SyntaxToken declaredIdentifier)
+        private LocalFunctionSymbol GetDeclaredLocalFunction(LocalFunctionStatementSyntax declarationSyntax)
```

Comment:
>GetDeclaredLocalFunction [](start = 36, length = 24)

The approach to adjusting symbols looks inconsistent. For a local we find the original symbol, then remap it. For a local function we are doing something different. I think we should have consistent approach for all local symbols (locals, local functions, parameters) and I am leaning towards approach used for locals.

Reply:
> The approach to adjusting symbols looks inconsistent. For a local we find the original symbol, then remap it. For a local function we are doing something different. I think we should have consistent approach for all local symbols (locals, local functions, parameters) and I am leaning towards approach used for locals.

This was marked as resolved, but I don't see any changes made.

---
In reply to: [327926167](https://github.com/dotnet/roslyn/pull/38769#discussion_r327926167) [](ancestors = 327926167)